google/cloud/forseti/common/gcp_api/_base_repository.py

```diff
@@ -15,12 +15,14 @@
 """Base GCP client which uses the discovery API."""
 import logging
 import threading
+import httplib2
```

Comment:
This sort was required by pylint, hence the change.

Reply:
Really?  That is really odd.  Looking deeper, `httplib2` is not a standard library, so it should not grouped here with the rest of the standard library.  Is there any reason why pylint is asking for this?  Just trying to understand it, so that I can apply it myself.

Reply:
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Can you tell me which item applies here?
The most relevant one that I see is this:
```
Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports

You should put a blank line between each group of imports.
```